src/spec-shutdown/dockerUtils.ts

```diff
@@ -324,7 +324,9 @@ export function dockerExecFunction(params: DockerCLIParameters | PartialExecPara
 }
 
 export async function dockerPtyExecFunction(params: PartialPtyExecParameters | DockerResolverParameters, containerName: string, user: string | undefined, loadNativeModule: <T>(moduleName: string) => Promise<T | undefined>): Promise<PtyExecFunction> {
-	const pty = await loadNativeModule<typeof ptyType>('node-pty');
+	const pty = !!process.env.COMPILE_FOR_PKG
```

Comment:
Please add a comment explaining that this env variable will be inlined at build time.